test/test_collection.py

```diff
@@ -182,6 +182,14 @@ def test_estimated_document_count(self):
         last_cmd_payload = self.explain.last_cmd_payload
         self._compare_command_dicts(last_cmd_payload, last_logger_payload)
 
+    def test_cli_tool(self):
+        import subprocess
+        res = subprocess.run(["python3",  "-m", "pymongoexplain",
+                              "./testscript.py"])
```

Comment:
Can we rename `testscript.py` to something more descriptive? Like `cli_test_script.py`. Having `testscript.py` in a folder called `test` is not at all suggestive of what the script might be for.

Reply:
Done.